--- Yoga.orignal.cpp	2018-05-22 21:24:15.002379898 +0100
+++ Yoga.cpp	2018-05-22 21:24:21.945791443 +0100
@@ -3781,7 +3781,7 @@
   config->experimentalFeatures[feature] = enabled;
 }
 
-inline bool YGConfigIsExperimentalFeatureEnabled(const YGConfigRef config,
+bool YGConfigIsExperimentalFeatureEnabled(const YGConfigRef config,
                                                  const YGExperimentalFeature feature) {
   return config->experimentalFeatures[feature];
 }
